packages/labs/analyzer/src/lib/lit-element/properties.ts

```diff
@@ -34,7 +35,18 @@ export const getProperties = (
 
   for (const prop of propertyDeclarations) {
     if (!ts.isIdentifier(prop.name)) {
-      throw new DiagnosticsError(prop, 'Unsupported property name');
+      analyzer.diagnostics.push(
+        createDiagnostic({
+          node: prop,
+          message:
+            '@lit-labs/analyzer only supports analyzing reactive properties of ' +
+            '`LitElement` extension classes named with plain identifiers. This ' +
```

Comment:
```suggestion
            '`LitElement` subclasses named with plain identifiers. This ' +
```

Also, it's a little ambiguous if the named things are the properties or the classes, but I couldn't think of a nice way to rephrase. No big deal.